dali/test/python/test_operator_readers_file.py

```diff
@@ -109,3 +110,22 @@ def test_file_reader_relpath_file_list():
             label = out_l.at(j)[0]
             index = 10000 - label
             assert contents == ref_contents(fnames[index])
+
+
+alias_batch_size=64
```

Comment:
I think that in general a smaller batch_size would do

Reply:
Having a batch that's larger than the dataset is a valuable test.